File: projects/go-sdk/handlers/getEvents.go
package handlers

import (
	"encoding/json"
	"go-sdk/utils"
	"net/http"
)

type getEventsQueryParams struct {
	ApiKey    string `json:"apiKey"`
	Region    string `json:"region"`
	RequestId string `json:"requestId"`
}

func GetEvents(w http.ResponseWriter, r *http.Request) {
	queryParams := getEventsQueryParams{
		ApiKey:    r.URL.Query().Get("apiKey"),
		Region:    r.URL.Query().Get("region"),
		RequestId: r.URL.Query().Get("requestId"),
	}

	client, auth := utils.InitSdk(queryParams.ApiKey, queryParams.Region)
	response := utils.ProcessResponse(client.FingerprintApi.GetEvent(auth, queryParams.RequestId))

	w.Header().Set("Content-Type", "application/json")
	w.WriteHeader(200)
	json.NewEncoder(w).Encode(response)
}

File: projects/python-sdk/handlers/v3/get_events.py
from flask import request

from handlers.v3.fingerprint_client import create_client, create_response


def get_events():
    request_id = request.args.get('requestId', '')
    api_instance = create_client()
    return create_response(lambda: api_instance.get_event_with_http_info(request_id))
